src/CloudTest.jl

```diff
@@ -165,15 +165,15 @@ publicPolicy(bucket) = """
 # use `with`, not `run`! if you `run`, it returns `conf, p`, where `p` is the server process
 # note that existing the Julia process *will not* stop the server process, which can easily
 # lead to "dangling" server processes. You can `kill(p)` to stop the server process manually
-function run(; dir=nothing, bucket=nothing, public=false, startupDelay=0.25, debug=false)
+function run(; dir=nothing, bucket=nothing, public=false, startupDelay=0.25, debug=false, bindIP="")
```

Comment:
should we just make `bindIP="127.0.0.1"` the default? I think that's generally how users are using this anyway.

Reply:
yes, sounds good to me! Thanks!